meson/meson.build

```diff
@@ -9,6 +9,9 @@ project(
 # libssp.  After recent MinGW runtime changes land, this should no longer be
 # necessary.
 subproject('ssp')
+subproject('uthash')
+subproject('check')
```

Comment:
Hmm, it'd be nice if `check` was optional so we didn't need to handle it here.  We don't run tests anyway.  (I realize I'm the one that [made it mandatory](https://github.com/ImagingDataCommons/libdicom/pull/23)...)

Reply:
PR in https://github.com/ImagingDataCommons/libdicom/pull/45.

Reply:
Nice! I merged this to main on my libdicom fork, I'll use that for this PR.